README.rst

```diff
@@ -11,4 +18,121 @@ warning threshold setting for failing the job and for parsing warnings from
 various tools and report them as errors.
 
 
+============
+Installation
+============
+
+Every release is uploaded to pip so it can be installed simply by using pip.
+
+.. code-block:: bash
+
+    # Python2
+    pip install mlx.warnings
```

Comment:
pip could also be mapped to pip3 (probably depends on the order in which they are installed?). Better to put pip2 instead.